identity-account/src/updates/update.rs

```diff
@@ -171,7 +172,7 @@ impl Update {
         let public: PublicKey = if let Some(method_private_key) = method_secret {
           insert_method_secret(storage, did, &location, type_, method_private_key).await
         } else {
-          storage.key_new(did, &location).await
+          storage.key_new(did, &location).await.map_err(|e| e.into())
```

Comment:
Nit: prefer `Into::into`.

```suggestion
          storage.key_new(did, &location).await.map_err(Into::into)
```